### ISSUE TITLE
Maintenance

### ISSUE THREAD

**Issue by reiven** [nlsdfnbch/hitbtc#30]
Looks like this is unmaintained, right?

**Comment by deepbrook** [comment]
Yes, pretty much. Between this, btfxwss, Pysher and bitex I simply don't have the time to support it. Would you like to pick up maintainership?
